c/common/src/codingstandards/c/initialization/GlobalInitializationAnalysis.qll

```diff
@@ -0,0 +1,93 @@
+import cpp
+import codingstandards.c.Objects
+import codingstandards.cpp.Concurrency
+import codingstandards.cpp.Type
+
+signature module GlobalInitializationAnalysisConfigSig {
+  /** A function which is not called or started as a thread */
+  default predicate isRootFunction(Function f) {
+    not exists(Function f2 | f2.calls(f)) and
```

Comment:
This will hold for functions that are called via function pointers.

Reply:
Good catch!